src/ui/component/publishForm/view.jsx

```diff
@@ -217,10 +217,12 @@ class PublishForm extends React.PureComponent<Props> {
       otherLicenseDescription,
       name: this.props.name || undefined,
       contentIsFree: this.props.contentIsFree,
-      price: this.props.price,
+      feeAmount: this.props.fee.amount,
```

Comment:
This should just be `fee`

Because if `fee` is undefined, it will error when trying to read `amount`